python/ray/rllib/evaluation/tf_policy_graph.py

```diff
@@ -102,12 +102,17 @@ def __init__(self,
         self._seq_lens = seq_lens
         self._max_seq_len = max_seq_len
         self._batch_divisibility_req = batch_divisibility_req
+
         self._optimizer = self.optimizer()
         self._grads_and_vars = [(g, v)
                                 for (g, v) in self.gradients(self._optimizer)
                                 if g is not None]
         self._grads = [g for (g, v) in self._grads_and_vars]
-        self._apply_op = self._optimizer.apply_gradients(self._grads_and_vars)
+        self._apply_op = self._optimizer.apply_gradients(
+            self._grads_and_vars,
+            global_step=self.global_step
+            if hasattr(self, "global_step") else None)
```

Comment:
Hm how about we replace this with `tf.train.get_or_create_global_step()`?

I would also add a note here that it is for TD3.